calcom/cal.com
# test: E2E Test for CalDav Calendar

## What does this PR do?
Adds an E2E Test for CalDav Calendar


Fixes #9372
/claim #9372




## Type of change




## How should this be tested?



- Are there environment variables that should be set?
  - `E2E_TEST_CALDAV_CALENDER_URL`
  - `E2E_TEST_CALDAV_CALENDAR_USERNAME`
  - `E2E_TEST_CALDAV_CALENDAR_PASSWORD`
  
- What is expected (happy path) to have (input and output)?
   - Run `yarn e2e caldav-calendar.e2e.ts`
   - It should successfully add the CalDav Calendar and test should pass
